phpstan.neon.dist

```diff
@@ -6,3 +6,4 @@ parameters:
         - public/
         - src/
         - tests/
+    checkGenericClassInNonGenericObjectType: false
```

Comment:
Better will be to use something like `// @phpstan-ignore`. I don't remember the exact annotation.

Reply:
I will add baseline file for project, to avoid this `@phpstan-ignore` in code.